lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -467,6 +467,10 @@ impl Dom {
 /// elements.
 #[derive(Clone, CloneRef, Debug)]
 pub struct DomLayers {
+    /// Bottom-most DOM scene layer with disabled panning. This layer is placed at the bottom
+    /// because mouse cursor is drawn on `canvas` layer, and would be covered by Welcome Screen
+    /// elements otherwise.
```

Comment:
Please, put similar description at `fullscreen_vis` layer. It is a back layer for the exactly same reason as the welcome screen.

Reply:
Done